### PR TITLE
cancelOrders - unification

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/htx.ts [hunk 1]
```diff
@@ -5701,7 +5701,7 @@ export default class htx extends Exchange {
         });
     }
 
-    async cancelOrders (ids, symbol: Str = undefined, params = {}) {
+    async cancelOrders (ids: string[], symbol: Str = undefined, params = {}): Promise<Order[]> {
```
Comment (carlosmiei) [inline ts/src/htx.ts:5704]: @ttodua you're updating the return type but we have to make sure the actual returned type matches, for instance here we just return the raw response 
image

File: ts/src/hyperliquid.ts [hunk 2]
```diff
@@ -1168,7 +1168,7 @@ export default class hyperliquid extends Exchange {
         return await this.cancelOrders ([ id ], symbol, params);
     }
```
Comment (carlosmiei) [inline ts/src/hyperliquid.ts:1170]: same here
image

File: ts/src/bingx.ts [hunk 3]
```diff
@@ -2674,7 +2674,7 @@ export default class bingx extends Exchange {
         return response;
     }
 
-    async cancelOrders (ids: string[], symbol: Str = undefined, params = {}) {
+    async cancelOrders (ids: string[], symbol: Str = undefined, params = {}): Promise<Order[]> {
```
Comment (carlosmiei) [inline ts/src/bingx.ts:2677]: same here 
image

**Comment by ttodua** [comment]
forgot to put this in draft at first, sorry... will ping you once ready

**Comment by ttodua** [comment]
too many changes are needed (to add individual exchange parsers) and in such bulk PR it's not viable. exchange specific PRs need to be made for each exchange.

**Pull request closed without merging**